book/src/ephemeral-local-volumes.md

```diff
@@ -38,12 +39,12 @@ Drivers must be modified (or implemented specifically) to support inline ephemer
 - Identity service
 - Node service
 
-### Feature gates
+### Feature gates (kubernetes version 1.15)
```

Comment:
Ah sorry, I would leave off updating the feature pages for the other features to @pohly and @gnufied because more things may need to be updated.  I just mean in the kubernetes-changelog.md, list those other features as well

Reply:
Seems like if they need "more" that could be added?  I'll go ahead an update in my morning if there's no feedback from @pohly and @gnufied

Reply:
Yes, just didn't want you to get stuck with updating the docs for other features too :-)  @pohly already has a PR out for ephemeral volumes https://github.com/kubernetes-csi/docs/pull/209

Reply:
Ok, I've removed the updates I added to expansion and ephemeral; just left the change-log entries.  Thanks!

Reply:
sorry about delay but thank you @j-griffith for updating it. lgtm